app/models/project_media.rb

```diff
@@ -485,6 +485,13 @@ def add_nested_objects(ms)
     ms.attributes[:requests] = requests
   end
 
+
+  def self.create_tags_in_background(**params)
+    project_media = ProjectMedia.find(params['project_media_id'])
```

Comment:
You should use `find_by_id` to avoid raising error in case item not found
i.e 
project_media = ProjectMedia.find_by_id params['project_media_id']
unless project_media.nil?
 .....

Reply:
@melsawy do we want to send a log or error to Sentry (if the tags are not created because the `project_media` is `nil`)?

Reply:
It's a good suggestion, agreed

Reply:
Added the `unless.project_media.nil?` and the `CheckSentry.notify`, question: should I keep it like this or would it be better to use `begin/rescue`?
https://github.com/meedan/check-api/blob/353612139eca53830ed062ac8612cc03b36a9a11/app/models/project_media.rb#L488-L498